tools/platyPSHelp/platyPSHelp.psm1

```diff
@@ -0,0 +1,582 @@
+    }
+
+    # Since we will be adding two <maml:para></maml:para> tags to the MAML, we need to adjust the size of the array
+    $newContentLength = $content.Length + (2 * $exampleCount)
+    $newContent = New-Object string[] $newContentLength
+
+    $buffer = 0
+
+    for ($idx = 0; $idx -lt $content.Length; $idx++)
+    {
+        $newContent[$idx + $buffer] = $content[$idx]
+
+        # If the next line is going to be the end of a remark in an example, add the two new lines for spacing
+        if ($content[$idx+1] -like "*</dev:remarks>*")
+        {
+            $newContent[$idx + $buffer + 1] = "<maml:para></maml:para>"
+            $newContent[$idx + $buffer + 2] = "<maml:para></maml:para>"
+
+            $buffer += 2
+        }
+    }
+
+    # Replace the contents of the current MAML with the new contents
+    $result = $newContent -join "`r`n"
+    $tempFile = Get-Item "$PathToCommandsFolder\$($MAML.Name)"
+
+    [System.IO.File]::WriteAllText($tempFile.FullName, $result)
+
+    # Check to ensure that the MAML file is still valid XML
+    [Reflection.Assembly]::LoadWithPartialName("System.Xml.Linq") | Out-Null
+    [System.Xml.Linq.XDocument]::Load($tempFile.FullName) | Out-Null
+}
+
+function Get-ServiceManagementDll
+{
+    [CmdletBinding()]
+    Param(
+        [Parameter(Mandatory=$True)]
+        [String]$Service,
+
+        [Parameter(Mandatory=$True)]
+        [String]$PathToAzure
+    )
+
+    $dlls = @{  
+                "Automation"       = "Automation\Microsoft.Azure.Commands.Automation.dll";
+                "Compute"          = "Compute\Microsoft.WindowsAzure.Commands.ServiceManagement.dll";
+                "HDInsights"       = "HDInsight\Microsoft.WindowsAzure.Commands.HDInsight.dll";
+                "ManagedCache"     = "ManagedCache\Microsoft.Azure.Commands.ManagedCache.dll";
+                "Networking"       = "Networking\Microsoft.WindowsAzure.Commands.ServiceManagement.Network.dll";
+                "Profile"          = "Services\Microsoft.WindowsAzure.Commands.Profile.dll";
+                "RecoveryServices" = "RecoveryServices\Microsoft.Azure.Commands.RecoveryServicesRdfe.dll";
+                "RemoteApp"        = "RemoteApp\Microsoft.WindowsAzure.Commands.RemoteApp.dll";
+                "Services"         = "Services\Microsoft.WindowsAzure.Commands.dll";
+                "Sql"              = "Sql\Microsoft.WindowsAzure.Commands.SqlDatabase.dll";
+                "StorSimple"       = "StorSimple\Microsoft.WindowsAzure.Commands.StorSimple.dll";
+                "TrafficManager"   = "TrafficManager\Microsoft.WindowsAzure.Commands.TrafficManager.dll"; 
+            }
+
+    if ($dlls.Keys -contains $Service)
+    {
+        return "$PathToAzure\$($dlls[$Service])"
+    }
+
+    throw "Unable to find dll for the given service."
+}
+
+function Get-ServiceManagementModuleName
+{
+    [CmdletBinding()]
+    Param(
+        [Parameter(Mandatory=$True)]
+        [String]$PathToModule
+    )
+
+    for (;;)
```

Comment:
Note that you can use the Get-Item commands to treat paths as files and extract properties, see: https://github.com/Azure/azure-powershell/blob/dev/tools/PublishModules.ps1#L183